examples/outreach/blogs/Blog_Estimators_DataSet.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
In general file names should be lowercase. Can we rename this?

Reply:
Done